client/js/_author/components/assessments/question_types/question_common/single_feedback.jsx

```diff
@@ -26,7 +26,7 @@ export default class SingleFeedback extends React.Component {
     *  to qbank. We make it here instead of in the editor, because the editor
     *  needs uploadScopeId in the select function.
     */
-    this.editorScopeId = _.get(this.props.feedback, 'answerId', guid());
```

Comment:
Because `_.get(thing, 'propertyThatIsNull', 'default')` will return `null`. It only sends the default back with `undefined`. The prop types yell about it.